### PR TITLE
Add LinkedIn logo with the link to the event

### PR DESCRIPTION
**Pull request by hharen**
Screenshot 2023-10-22 at 04 19 54

### CODE REVIEW

File: src/2023/index.html [hunk 1]
```diff
@@ -378,13 +378,21 @@ <h2>Partners</h2>
...
+                </div>
+              </a>
             </div>
           </div>
         </div>
 
         <div>
           <p class="small italic">
-            Last updated on <time>20th October 2023</time>.
+            Last updated on <time>22th October 2023</time>.
```
Comment (crackofdusk) [inline src/2023/index.html:395]:
```suggestion
            Last updated on <time>22nd October 2023</time>.
```

**Review by crackofdusk — APPROVED** [review]

File: src/2023/index.html [hunk 2]
```diff
@@ -378,13 +378,21 @@ <h2>Partners</h2>
                   </svg>
                 </div>
               </a>
+
+              <a href="https://www.linkedin.com/events/helveticruby20237121673048570634240/comments/" title="Follow us on LinkedIn">
```
Comment (lxxxvi) [inline src/2023/index.html:382]: I don't have LinkedIn. The link takes me to a "Page not found", I expected it to be "You need to sign in". Can you confirm that the link is valid?

Comment (hharen) [reply src/2023/index.html]: Ridiculous LinkedIn, I tried it yesterday, and I copied the link from a different place (the original share button) today and of course I tested it. It doesn't seem to work reliably. Will look into it. 

Can you try this one in the meantime? https://www.linkedin.com/events/helveticruby20237121673048570634240/

Comment (crackofdusk) [reply src/2023/index.html]: I see this with the new link:

image

Comment (lxxxvi) [reply src/2023/index.html]: I get the same.

**Review by lxxxvi — APPROVED** [review]

File: src/2023/index.html (hunk 2 above)
Comment (hharen) [reply src/2023/index.html]: Nice.

**Pull request merged**